README.md

```diff
@@ -94,3 +94,15 @@ This decision was made to avoid errors in our apps when accidentally using test
 ## Code of conduct
 
 For guidelines regarding the code of conduct when contributing to this repository please review [https://www.dabapps.com/open-source/code-of-conduct/](https://www.dabapps.com/open-source/code-of-conduct/)
+
+### Import resolvers
+
+If your project uses a custom module resolver (e.g. aliasing `^` to the project root), you will need to define a rule for the import plugin, to tell it how to resolve modules.
+
+See https://github.com/benmosher/eslint-plugin-import#resolvers
+
+```
+{
+  "import/resolver": "./my-resolver"
```

Comment:
This should go in by default.

Reply:
Yeah this is an artifact from when you did this library so I hope you blame yourself immensely. I'm going to try set this up with a project that uses absolute paths and see what happens.

Reply:
I've updated the readme, there are still cases (wethrive for example) where we're using something other than webpack for path resolving, so think it'll still be good to leave in here just in case.

Reply:
Will add in next major release. 👍